config/authorization/config.ex

```diff
@@ -79,6 +79,11 @@ defmodule Acl.UserGroups.Config do
                         "https://data.vlaanderen.be/ns/mobiliteit#Verkeersbordcategorie",
                         "https://data.vlaanderen.be/ns/mobiliteit#Verkeersbordconcept",
                         "https://data.vlaanderen.be/ns/mobiliteit#VerkeersbordconceptStatus",
+                        # For logs, reports (dashboard and report service)
+                        "http://persistence.uni-leipzig.org/nlp2rdf/ontologies/rlog#Entry",
```

Comment:
hmm should these be publicly accessible?

Reply:
Just copying from loket here ([see here](https://github.com/lblod/app-digitaal-loket/blob/master/config/authorization/config.ex#L123)) and I'm not fluent in mu-auth config yet. It does mention as a TODO to not put a lot of that in public.
So what user role(s) should these entries be allocated to? Writer, publisher, signer, ... or should we create a new role? Do we make a new login mechanism for these users?